src/utils/apiCalls.js

```diff
@@ -14,7 +14,10 @@ export function getTodaysPrice(ticker) {
 }
 
 export function getYesterdaysPrice(ticker) {
```

Comment:
Does this function belong in dateUtils? Seems to introduce ticker stuff, which is hardly date-related.

Reply:
this is in apiCalls, not dateUtils.